www/templates/control/main-control.html

```diff
@@ -29,20 +29,33 @@
       </label>
     </div>
     <div class="control-list-item">
-      <div class="control-list-text">Force sync</div>
-      <div ng-click="forceSync()" ng-style="getIconButtonStyle('D6C780')"><i class="ion-android-sync" ng-style="getIconStyle()"></i></div>
-    </div>   
+      <div class="control-list-text">Low accuracy</div>
+      <label ng-click="willUseLowAccuracy()" class="toggle control-list-toggle">
```

Comment:
This one I actually need to call BEMDataCollection stuff to update the config. I could, of course, bind the ng-model to a setter function, but then it would be the same to this function. For the toggle in dark theme, I also need to have a function call for appPreference fetching.